### PR TITLE
fix(kubelet, logs): change format specifier

### PR DESCRIPTION
**Pull request by utkarshmani1997**
Signed-off-by: utkarshmani1997 <utkarshmani1997@gmail.com>

**What type of PR is this?**
/kind bug

**What this PR does / why we need it**:
I have found that the kubelet was printing the logs in []bytes, which is
not readable, this is happening since `%+v` is being used. This PR will fix
the logs.
Output from the logs: 
```
Failed with err write tcp 10.200.0.85:10250->10.200.0.87:58918: write: connection       reset by peer when writing log for log file "/var/log/pods/d08ee327-5ef3-11e9-a7cf-525400451066/nginx-ingress-controller/0.log": &{timestamp:{wall:361813941 ext      :63690998724 loc:<nil>} stream:stderr log:[50 48 49 57 47 48 52 47 49 54 32 48 56 58 48 53 58 50 52 32 91 105 110 102 111 93 32 52 55 50 51 35 52 55 50 51 58 32       42 51 50 50 48 57 51 57 32 91 108 117 97 93 32 98 97 108 97 110 99 101 114 46 108 117 97 58 55 50 58 32 115 121 110 99 95 98 97 99 107 101 110 100 40 41 58 32 11      6 104 101 114 101 32 105 115 32 110 111 32 101 110 100 112 111 105 110 116 32 102 111 114 32 98 97 99 107 101 110 100 32 105 110 116 101 114 119 97 118 101 45 54       50 55 45 99 109 45 97 99 109 101 45 104 116 116 112 45 115 111 108 118 101 114 45 119 119 54 113 122 45 56 48 56 57 46 32 83 107 105 112 112 105 110 103 46 46 4      6 44 32 99 111 110 116 101 120 116 58 32 110 103 120 46 116 105 109 101 114 10]}
```
Go playground link: https://play.golang.org/p/vQHmQlwy6qr

**Does this PR introduce a user-facing change?**:
```
NONE
```

/assign @utkarshmani1997

### CODE REVIEW

**Comment by utkarshmani1997** [comment]
I signed it

**Review by mattjmcnaughton — APPROVED** [review]
/lgtm

Just for my own curiosity, would you be able to post examples of log messages before and after the change?

**Comment by mattjmcnaughton** [comment]
@utkarshmani1997, thanks for your diff!

Two quick things we need before moving forward: we need you to sign the CLA (you can find instructions here) and you need to add a release note (which I think is as easy as adding
```
NONE
```
to your pr body.

**Comment by mattjmcnaughton** [comment]
Please let me know if you have any questions :)

**Comment by utkarshmani1997** [comment]
Thanks @mattjmcnaughton for the review, i have updated PR description with the logs before and after.
I am having issues with cla, i have already signed the cla but it is still showing that cla failed.

/retest

**Comment by utkarshmani1997** [comment]
"I signed it"

File: pkg/kubelet/kuberuntime/logs/logs.go [hunk 1]
```diff
@@ -381,7 +381,7 @@ func ReadLogs(ctx context.Context, path, containerID string, opts *LogOptions, r
 				klog.V(2).Infof("Finish parsing log file %q, hit bytes limit %d(bytes)", path, opts.bytes)
 				return nil
 			}
-			klog.Errorf("Failed with err %v when writing log for log file %q: %+v", err, path, msg)
+			klog.Errorf("Failed with err %v when writing log for log file %q: %q", err, path, msg)
```
Comment (prateekpandey14) [inline pkg/kubelet/kuberuntime/logs/logs.go]: govet check failed with error  " `Errorf format %q has arg msg of wrong type`" @utkarshmani1997

Comment (utkarshmani1997) [reply pkg/kubelet/kuberuntime/logs/logs.go]: Since the purpose of msg is to print the logs, shall i only print the string(msg.log) ?

**Comment by utkarshmani1997** [comment]
@mattjmcnaughton Please review, i have incorporated changes

**Pull request closed without merging**